posts/announce-lts-21-nightly-ghc9.6.md

```diff
@@ -0,0 +1,24 @@
+---
+author: Jens Petersen and Andreas Ländle
+title: LTS 21 release for ghc-9.4 and Nightly now on ghc-9.6
+description: Stackage LTS 21 (ghc-9.4.5) has been released and Stackage Nightly moves to ghc-9.6.2.
+timestamp: 2023-06-24T20:00:00Z
+---
+
+## Stackage LTS 21 has been released
+
+The Stackage team is very happy to announce the first [Stackage LTS version 21](https://www.stackage.org/lts-21.0) snapshot has been released this week, based on GHC stable version 9.4.5.
+
+LTS 21 includes many [package changes](https://www.stackage.org/diff/lts-20.26/lts-21.0), and again has over 3000 packages!! Thank you for all the nightly contributions that made this possible.
+
+If your package is missing from LTS 21 and builds there, you can easily request to have it added using our straightforward process: just open a github issue in the [lts-haskell](https://github.com/commercialhaskell/lts-haskell/issues) project and following the simple steps in the template.
+
+## Stackage Nightly updated to ghc-9.6.2
+
+At the same time we are also excited to have moved [Stackage Nightly](https://www.stackage.org/nightly) already to GHC 9.6.2: the initial snapshot being [nightly-2023-06-23](https://www.stackage.org/nightly-2023-06-23)!  Current nightly has almost 2449 packages, but we expect that number to continue to grow over the coming days, weeks, and months: we very much welcome your contributions and help with that.
+
+The main reasons for the having to disable packages that are now in lts-21 being changes to exports for monad functions and optparse-applicative prettyprinting, and also the openFD API change. Thanks to all those who have already done work updating their packages to ghc-9.6.
```

Comment:
Very convoluted sentence, should probably be split up.  What about something like:

> Common reasons for disabling packages were:
> * unix-2.8 changed the API of `openFd`
> * mtl-2.3 stopped reporting from e.g. `Control.Monad`
> * optparse-applicative changed API